sda-auth/config.go

```diff
@@ -172,16 +174,12 @@ func (c *Config) readConfig() error {
 		log.Printf("Setting log level to '%s'", stringLevel)
 	}
 
-	if viper.GetString("s3Inbox") == "" {
-		return fmt.Errorf("%s not set", "s3Inbox")
-	}
-
```

Comment:
```
	for _, s := range []string{"s3Inbox", "publicFile"} {
		if viper.GetString(s) == "" {
			return fmt.Errorf("%s not set", s)
		}
	}
```

Reply:
Need to ensure all mandatory variables are set.

Reply:
[x]